packages/roosterjs-editor-types/lib/interface/DefaultFormat.ts

```diff
@@ -36,6 +36,16 @@ interface DefaultFormat {
      * Whether has underline
      */
     underline?: boolean;
+
+    /**
+     * original source background, used for converting back to light mode
+     */
+    ogsb?: string;

```

Comment:
like I said above these would be nicer as just `originalSourceBackground` or `lightmodeBackgroundColor` -- descriptive names onegai